new-pipeline/dist.html

```diff
@@ -220,6 +301,7 @@ <h3><select id="selected-key4" class="multiselect" title="Selected key"></select
     <script src="../v2/telemetry.js"></script>
     <script src="src/dashboards.js"></script>
     <script src="src/dist.js"></script>
+    <script src="src/evo.js"></script>
```

Comment:
These scripts interfere with each other by overriding each others' global storage. I'm not sure dist.js and evo.js can be present in the same context in the way they are written today.